### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Problem] Developers need information how to use project
[Solution] Add README.md to github project

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,73 @@
-# tau
-TAU (Tizen Advanced UI) project for supporting web app development based on web-based ui widget components library.  This will be replacement and clone of previously existing https://review.tizen.org/gerrit/#/admin/projects/platform/framework/web/tau
+# TAU
+
+The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components. The components represent a visual UI element, such as a button or slider, which gives you interaction and manipulation features.
+
```
Comment (lslachciak) [inline README.md]: I suggest adding a Getting Started section name according to the
https://gist.github.com/PurpleBooth/109311bb0361f32d87a2
I love this example as a  good Readme file

Comment (TomaszLukawskiSam) [reply README.md]: I modified PR. I left several section marked as to do. File AUTHOR needs update. Project has two licenses files MIT & FLORA.

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Copyright (c) 2013 - 2014 Samsung Electronics Co., Ltd. All rights reserved.
```
Comment (lslachciak) [inline NOTICE]: 2013-2019

File: AUTHOR [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Koeun Choi <koeun.choi at samsung dot com>
```
Comment (lslachciak) [inline AUTHOR]: Can we just put
Samsung Electronics Co., Ltd.
like google is doing in projects:
https://github.com/google/amber/blob/master/AUTHORS

File: COPYING [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+by the OSI at opensource.org/licenses/MIT), except for
+following files:
+	*demos/MobileWinset/src/tips/pinch-event/pinch.js
+	*src/js/core/event/pinch.js
+	*src/js/profile/mobile/widget/mobile/Gallery3d.js
+
+****************************************************************************
+The MIT License (MIT)
+
+Copyright (c) 2000 - 2014 Samsung Electronics Co., Ltd.
```
Comment (lslachciak) [inline COPYING]: 2019

File: README.md [hunk 5]
```diff
@@ -1,2 +1,115 @@
...
+`npm install http-server -g`
+
+- Launch server in app directory:
+
+`http-server -p <port> -a localhost`
+
+eg. `http-server -p 8888 -a localhost`
+
+
+### And coding style tests
```
Comment (lslachciak) [inline README.md]: "And" is not needed here, just write
"Coding style tests"

File: README.md [hunk 6]
```diff
@@ -1,2 +1,115 @@
-# tau
-TAU (Tizen Advanced UI) project for supporting web app development based on web-based ui widget components library.  This will be replacement and clone of previously existing https://review.tizen.org/gerrit/#/admin/projects/platform/framework/web/tau
+# TAU
+
+The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components. The components represent a visual UI element, such as a button or slider, which gives you interaction and manipulation features.
```
Comment (lslachciak) [inline README.md]: grammar: 
The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components.

TAU **components** allow to create and manage **components**?
or TAU contains components...

File: README.md [hunk 7]
```diff
@@ -1,2 +1,115 @@
...
+@todo
+
+## Authors
+
+* Authors - [AUTHOR]
+
+See also the list of [contributors](https://github.com/SAMSUNG/TAU/contributors) who participated in this project.
+
+## License
+
```
Comment (lslachciak) [inline README.md]: this section duplicates function of COPYING file which states 
This software is licensed under the MIT license (as defined
by the OSI at opensource.org/licenses/MIT), except for
following files:...
How about refering to this file and stating that 

this project is licensed under MIT except some files under Flora license. For details see COPYING...

File: NOTICE (hunk 2 above)
Comment (TomaszLukawskiSam) [reply NOTICE]: Done

File: README.md (hunk 5 above)
Comment (TomaszLukawskiSam) [reply README.md]: Done

File: COPYING (hunk 4 above)
Comment (TomaszLukawskiSam) [reply COPYING]: Done

File: README.md (hunk 7 above)
Comment (TomaszLukawskiSam) [reply README.md]: Sounds better... thank you.

File: README.md (hunk 6 above)
Comment (lslachciak) [reply README.md]: agreed version: The TAU library contains components which allows you to create Web Application UI. Each components represents UI element, such as a...,

**Pull request merged**